### PR TITLE
Discord webhook

### PR DESCRIPTION
**Pull request by ycanardeau**

### CODE REVIEW

File: VocaDbModel/Utils/AppConfig.cs [hunk 1]
```diff
@@ -239,5 +239,9 @@ public static SongType[] SongTypes
 		public static string YoutubeApiKey => Val("YoutubeApiKey");
 
 		public static string VimeoApiKey => Val("VimeoApiKey");
+
+		public static string DiscordWebhookUrl = Val("DiscordWebhookUrl");
```
Comment (riipah) [inline VocaDbModel/Utils/AppConfig.cs:243]: Might be better to use "=>" here, same as others, so we don't need to read this for tests.

Comment (riipah) [reply VocaDbModel/Utils/AppConfig.cs:243]: And maybe add some comments here explaining the purpose

Comment (ycanardeau) [reply VocaDbModel/Utils/AppConfig.cs]: Oh, thanks. This is due to my new keyboard.

**Comment by ycanardeau** [comment]
@riipah I moved the code to `DiscordWebhookNotifier` because I realized that sending notifications should be optional.

**Pull request merged**